core/templates/dev/head/pages/base.html

```diff
@@ -94,6 +94,7 @@
   </head>
 
   <body>
+    <service-bootstrap></service-bootstrap>
```

Comment:
Since base is not used anymore in the static pages should this also be defined there?

Reply:
Yes, it should be. Although the service is not used there but it would be safe to trigger Angular there too should the service be used there.
Done!